k8s/namespaces/rd/rd-professional-api/aat.yaml

```diff
@@ -0,0 +1,11 @@
+apiVersion: helm.fluxcd.io/v1
+kind: HelmRelease
+metadata:
+  name: rd-professional-api
+spec:
+  values:
+    java:
+      environment:
+        IDAM_URL: https://idam-api.aat.platform.hmcts.net
+        CCD_URL: http://ccd-user-profile-api-aat.service.core-compute-aat.internal
+        DUMMY_URL: https://idam-api.aat.platform.hmcts.net
```

Comment:
you can remove dummy url.. it was added only to trigger the flux deployment

Reply:
Removed dummy URL